### Null Hypothesis (H0): There is no association between device type and customer satisfaction.

### Alternate Hypothesis (H1): There is an association between device type and customer satisfaction.

In [1]:
import numpy as np
import scipy.stats as stats

In [2]:
df = np.array([
    [50, 70],
    [80, 100],
    [60, 90],
    [30, 50],
    [20, 50]
])

In [3]:
stats.chi2_contingency(df)

Chi2ContingencyResult(statistic=np.float64(5.638227513227513), pvalue=np.float64(0.22784371130697176), dof=4, expected_freq=array([[ 48.,  72.],
       [ 72., 108.],
       [ 60.,  90.],
       [ 32.,  48.],
       [ 28.,  42.]]))

chi2_stat = 5.6382

p_value = 0.2278

dof = 4

alpha = 0.05

p_value > alpha

0.2278 > 0.05

#### Fail to reject the null hypothesis. There is no association between device type and customer satisfaction.